src/android/MainActivity.java

```diff
@@ -141,62 +104,38 @@ public static String excutePost(String targetURL, String urlParameters)
         return "Data Sent";
     }
 
-    @Override
-    public void onLocationChanged(Location location) {
-
-        lat = location.getLatitude();
-        lon = location.getLongitude();
-        alt = location.getAltitude();
-
-        SimpleDateFormat GPStime = new SimpleDateFormat("yyyyMMdd'T'HH:mm:ss");
-        timestamp = GPStime.format (location.getTime());
-    }
-
-    @Override
-    public void onStatusChanged(String s, int i, Bundle bundle) {
-
-    }
-
-    @Override
-    public void onProviderEnabled(String s) {
-
-    }
-
-    @Override
-    public void onProviderDisabled(String s) {
-    }
 
     private class HttpAsyncTask extends AsyncTask<String, Void, String> {
         @Override
         protected String doInBackground(String... urls) {
 
             JSONObject jObject;
-            JSONArray jArray_wifi, jArray_bte;
+            JSONArray jArray_wifi, jArray_bte, jArray_gps;
 
-            w_dbManager = new DBHelper(context, "WiFi.db", null, 1);
+            w_dbManager = DBHelper.getInstance(context, "WiFi.db", null, 1);
```

Comment:
as noted above, all x_dbManager instances will be the same instead of different databases.